revocation/result/results.go

```diff
@@ -74,7 +78,7 @@ type ServerResult struct {
 
 	// RevocationMethod is the method used to check the revocation status of the
 	// certificate, including Unknown(0), MethodOCSP(1), MethodCRL(2)
-	RevocationMethod int
+	RevocationMethod revocation.Method
```

Comment:
Here you should reference the exported type in an exported type/method.
```suggestion
	RevocationMethod Method
```

Reply:
However, we cannot call `result.go` is in `result` package and cannot call `Method`.

Reply:
Refactored and moved the `revocation.Method` to `result.RevocationMethod`